src/sentry/conf/server.py

```diff
@@ -3098,3 +3098,7 @@ def build_cdc_postgres_init_db_volume(settings):
 
 # Set the URL for signup page that we redirect to for the setup wizard if signup=1 is in the query params
 SENTRY_SIGNUP_URL = None
+
+# Temporary allowlist for specially configured organizations to use the direct-storage
+# driver.
+SENTRY_REPLAYS_STORAGE_ALLOWLIST = []
```

Comment:
quick note: let's remember to update this value in `getsentry` when we go to do so.